Volume_Coupling/Volume_Coupling.H

```diff
@@ -0,0 +1,61 @@
+#ifndef Volume_Coupling_H
+#define Volume_Coupling_H
+
+#include "Interface.H"
+
+#include "Volume_Coupling/Generic_volScalarField.H"
+#include "Volume_Coupling/Generic_volVectorField.H"
+
```

Comment:
These includes are still missing.

Reply:
Yes, will be pushing things soon.
I should have named the PR with Draft as I was still working on cleaning up few things.

Reply:
I converted the PR to draft, mark as "ready for review" when you are ready. :-)